Ev/Ev.php

```diff
@@ -387,14 +389,20 @@ final class Ev
 
 /**
  * Class EvWatcher
- *
- * @property-read bool $is_active TRUE if the watcher is active. FALSE otherwise.
- * @property-read bool $is_pending TRUE if the watcher is pending, i.e. it has outstanding events, but its callback
- *      has not yet been invoked. FALSE otherwise. As long, as a watcher is pending (but not active), one must not
- *      change its priority.
  */
 abstract class EvWatcher
 {
+    /**
+     * @var bool TRUE if the watcher is active. FALSE otherwise.
+     */
+    #[Immutable] public $is_active;
```

Comment:
PHP 7 will skip the property definition because `#` means comment. Shouldn't the attribute definition and the property definition be split to two lines?

Reply:
@b1rdex phpstorm-stubs are not meant to be executed by PHP, it serves mostly static analysis purpose.